### DISCUSSION TITLE
High Distance Snap allows for out-of-bounds hit object placement

### DISCUSSION THREAD

**Discussion by Walavouchey** [ppy/osu#18229] [Q&A] [answered]
the video below should explain the situation clearly:

https://user-images.githubusercontent.com/36758269/167931995-1ef8f222-f0e9-45f6-b00e-5c702e0a6090.mp4

you can of course place things offscreen with the same technique too:

https://user-images.githubusercontent.com/36758269/167933651-75ef8f73-d4d6-459c-afbb-22a753d08e6f.mp4

version: latest release (2022.509.0-lazer)

logs: n/a

**Comment by bdach** [comment, marked as answer]
#18231
